apps/explorer/src/components/transaction-card/RecentTxCard.tsx

```diff
@@ -162,16 +136,48 @@ function LatestTxCard({ ...data }: RecentTx) {
         [setSearchParams]
     );
 
+    const countQuery = useQuery(['transactions', 'count'], () => {
```

Comment:
should we add network in the key?

Reply:
The query client is reset whenever the network changes, so these don't need the network. I opted to go this route to make it easier to develop against, and to make the cache easier to reason about especially around network changes.